docs/dynamic_batch_size.md

```diff
@@ -0,0 +1,195 @@
+# Dynamic batch size with OpenVINO&trade; Model Server Demultiplexer
+
+## Intoduction
+This document guides how to configure DAG Scheduler pipeline to be able to send predict request with arbitrary batch size without model reloading.
+
+With OpenVINO&trade; Model Server Demultiplexing infer request sent from client application can have various batch sizes and changing batch size does not require model reload.
+
+More information about this feature can be found in [dynamic batch size in demultiplexing](demultiplexing.md#dynamic-batch-handling-with-demultiplexing)
+
+*Note:* When using `demultiply_count` parameters, only one demultiplexer can exist in pipeline.
+
+- Example client in python grpc_serving_client.py can be used to request the pipeline. Use `--dag-batch-size-auto` flag to add additional dimension to the input shape which is required for demultiplexing feature.
+
+- The example uses model [resnet](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/resnet50-binary-0001/README.md).
+
+- While using resnet model with grpc_serving_client.py the script processes the output from the server and displays the inference results using previously prepared file with labels. Inside this file each image has assigned number, which indicates the correct recognition answer.  
+
+## Steps
+Clone OpenVINO&trade; Model Server github repository and enter `model_server` directory.
+#### Download the pretrained model
+Download model files and store it in `models` directory
+```Bash
+mkdir -p models/resnet/1
+curl https://download.01.org/opencv/2021/openvinotoolkit/2021.1/open_model_zoo/models_bin/1/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin https://download.01.org/opencv/2021/openvinotoolkit/2021.1/open_model_zoo/models_bin/1/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml -o models/resnet/1/resnet50-binary-0001.bin -o models/resnet/1/resnet50-binary-0001.xml
```

Comment:
new location for the models is in https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.4/models_bin/2/resnet50-binary-0001/FP32-INT1/